chia/cmds/plotnft.py

```diff
@@ -121,13 +121,6 @@ def create_cmd(
     required=True,
     callback=validate_fee,
 )
-@click.option(
-    "--fee",
-    help="Fee Per Transaction, in Mojos. Fee is used TWICE: once to leave pool, once to join.",
-    type=int,
-    callback=validate_fee,
-    default=0,
-)
```

Comment:
This diff suggests we're removing the fee parameter rather than adding it.  That's not right right?

Reply:
Good eye! This removes a duplicate flag declaration that snuck into main earlier, so this is okay.